docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -1355,81 +1370,168 @@ intervention. The food vehicle referred to in the table is wheat flour.
     - 13.8 (11.5, 16.1)
     - 14.2 (11.8, 16.5)
 
-**For Ethiopia**, assume
+The above data for India and Nigeria is from Table 4 in [Aaron-et-al-2017]_. The
+numbers for Ethiopia were chosen so that (i) the mean of existing wheat flour
+fortification coverage is close to 0, based on
+[GFDx-Ethiopia-Fortification-Dashboard]_, (ii) the percentage of people eating
+wheat flour is 28%, based on [Ethiopian-Federal-Ministry-of-Health-2011]_, (iii)
+between 53% and 55% of the wheat flour is fortifiable, based data from
+[Ethiopian-Federal-Ministry-of-Health-2011]_ and
+[GFDx-Ethiopia-Fortification-Dashboard]_, and (iv) the 95% confidence intervals
+for :math:`b` and :math:`c` have a width of 10% (chosen arbitrarily).
+
+Marginal distributions of :math:`a`, :math:`b`, and :math:`c`
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+**For Ethiopia parameter** :math:`a`, assume
 
 .. math::
 
-  a_\textit{Ethiopia} \sim \operatorname{Beta}(0.1,9.9),\quad
-  b_\textit{Ethiopia} \sim \operatorname{Beta}(0.5,3.1),\quad
-  c_\textit{Ethiopia} \sim \operatorname{Beta}(0.8,2.36).
-
-The means of these `Beta distributions
-<https://en.wikipedia.org/wiki/Beta_distribution>`_ will have the values shown
-in the table. Each of the densities has an asymptote at 0 and an x-intercept at
-1, and the parameters :math:`\alpha` and :math:`\beta` were chosen to vary
-monotonically with the mean. The numbers for Ethiopia were chosen so that (i)
-the mean of existing fortification coverage is close to 0, (ii) the percentage
-of people eating wheat flour is similar to that in Nigeria, and (iii) 55%
-of the wheat flour is fortifiable, based on the `Global Fortification Data
-Exchange <https://tinyurl.com/rdm4wza>`_.
+  a_\textit{Ethiopia} \sim \operatorname{Beta}(0.1,9.9).
+
+This `Beta distribution <https://en.wikipedia.org/wiki/Beta_distribution>`_ will
+have mean 1.0 as in the table. The density has an asymptote at 0 and an
```

Comment:
mean of 0.1, right?

Reply:
Oh, I meant 1.0 percent, so actually a mean of 0.01.  I'll clarify.